src/udp/connection_cookie.rs

```diff
@@ -0,0 +1,214 @@
+use std::net::SocketAddr;
+
+use aquatic_udp_protocol::ConnectionId;
+
+use crate::protocol::clock::SinceUnixEpoch;
+use crate::udp::ServerError;
+
+pub fn make_connection_cookie(remote_address: &SocketAddr) -> ConnectionId {
+    let period = detail::get_period_now();
+
+    let cookie = detail::make_cookie_internal(remote_address, &period);
```

Comment:
@da2ce7 can we move period generation inside?:

```rust
let cookie = detail::make_cookie_for_current_period(remote_address)
```

On the other hand, maybe we can rename the `detail` mod to something like:

```rust
let cookie = cookie_builder::make_for_current_period(remote_address)
```

Reply:
@josecelano Thanks, and done!